frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -80,50 +80,44 @@ const MatchItemWrapper = withStyles({
    }
 })(Card);
 
-export const MatchItem = (props: MatchItemProps,) => {
-
-   const isWinnerTeam1 = props.winnerTeamId === props.team1.teamId;
-   const isWinnerTeam2 = props.winnerTeamId === props.team2.teamId;
-
-   const isMatchFinished = props.matchStatus === MatchStatus.FINISHED;
-   const isMatchStarted = props.matchStatus === MatchStatus.STARTED;
+export const MatchItem = (
+   {
+      expanded,
+      handleChangeExpander,
+      level,
+      matchId,
+      matchNumber,
+      matchStatus,
+      onClickTeam,
+      team1,
+      team2,
+      winnerTeamId
+   }: MatchItemProps,) => {
 
-   const agreeDialogCallback = (matchId: string, teamName: string): void => props.onClickTeam(matchId, teamName);
+   const isWinnerTeam1 = winnerTeamId === team1.teamId;
+   const isWinnerTeam2 = winnerTeamId === team2.teamId;
 
+   const isMatchFinished = matchStatus === MatchStatus.FINISHED;
```

Comment:
Te zmienne sa w ogóle nie używane. IntelliJ to nawet pokazuje :) 
![image](https://user-images.githubusercontent.com/18637116/114698751-bf105e00-9d1f-11eb-8b92-9ed5d16bd0db.png)